data/murderrun/function/traps/levitation_trap/levitation_trap.mcfunction

```diff
@@ -0,0 +1,8 @@
+# Particle
+	particle minecraft:falling_dust{block_state:{Name:white_concrete}} ~ ~ ~ 0.3 0.3 0.3 0 1 force @a[distance=..20]
```

Comment:
Can you change this to a purple block? Probably do like purple concrete or something

Reply:
No

Reply:
ok bro